projects/fsastorefrontstyles/scss/root.scss

```diff
@@ -3,6 +3,10 @@
 :root {
   @each $color, $value in $theme-colors {
     --cx-color-#{$color}: #{$value};
+
+    .fsa-bg-#{$color} {
+      background-color: $value;
+    }
```

Comment:
Please remove this from root. We have the same function for borders in `fs-borders.scss`, maybe we can unite the two but move it to `building-blocks.scss` or `genereal.scss`?